src/cli/run.ts

```diff
@@ -12,9 +12,12 @@ import { archLogger } from "../logger/chalk-theme";
       process.exit(result.exitCode);
     }
   } catch (err) {
-    archLogger.error("cli runtime exception: " + err, true);
+    archLogger.error("cli runtime exception: " + err, {
+      logTimestamp: true,
+      sendNotification: true,
+    });
```

Comment:
Do we need to `await` any of these `archLogger.error` calls, now that is it an `async` function? for example, in this function, could the process exit before the notification is sent (due to the syncronous call?)

Reply:
Oh my goodness, good catch. It was in mind and I totally forgot to do that. Thanks!